Week1/homework/exercise1.js

```diff
@@ -0,0 +1,10 @@
+console.log("Hello, world! // English")
+console.log("Oi, mundo! // Portuguese")
+console.log("Hola, mundo! // Spanish")
+console.log("Halo, dunia! // Indonesian")
+console.log("Ciao, mondo! // Italian")
+console.log("Hallo, wereld! // Nederlandse")
+console.log("Merhaba, dunya! // Turkish")
+console.log("Bonjour, le monde! // French")
+console.log("Hej, verden! // Denmark")
+console.log("Geia, kósmo! // Greek")
```

Comment:
👍 but I think you meant to have the comments (`// English`) outside the `console.log`, right?